### DISCUSSION TITLE
Firefox Bad Memory Initial Size

### DISCUSSION THREAD

**Discussion by tbuchs** [emscripten-core/emscripten#22486] [General]
Hi, I am building an application for a few months now and everything goes well. However, since I use Memory64 I am limited to FIrefox Nightly or Chrome. Since the latest Firefox Nightly Update (as far as I can tell), I always get ``` Uncaught TypeError: bad Memory initial size ``` when opening the page. I don't get that in Chrome and also if I use older builds of my application, which worked in Firefox Nightly, it shows that error. 

Since I use some features, which only work well currently in Firefox Nightly, I really would like to use it over Chrome. Do you have any idea, what the reason for this error could be?

**Comment by tbuchs** [comment]
and with the old version of Firefox, it works as well

**Reply by kripken** [reply]
This sounds like a change in Firefox Nightly. If you think it is a bug, I would open an issue there.

**Reply by sbc100** [reply]
I think this is likely due to a change in the spec: https://github.com/WebAssembly/memory64/issues/68

I'm working on change to make emscripten output compatible with this.

**Reply by sbc100** [reply]
Can  you confirm, are you using imported memory?  (Or perhaps dynamic linking, which implies imported memory?)

**Reply by tbuchs** [reply]
Yes, I am using dynamic linking

**Comment by sbc100** [comment]
This was fixed in #22497
